### PR TITLE
Update 4.4 branch with 4.3 branch

### PR DESCRIPTION
**Pull request by rauldpm**
This PR update the 4.4 branch with the latest changes of 4.3 branch

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +11,11 @@ All notable changes to this project will be documented in this file.
 - Remove references to backup/groups [#1299](https://github.com/wazuh/wazuh-packages/pull/1299)
 - Add Azure's orm file to Solaris 11 SPEC [#1453](https://github.com/wazuh/wazuh-packages/pull/1453)
 
+## [v4.3.1]
```
Comment (okynos) [inline CHANGELOG.md:14]: Add the rest of the versions

File: aix/SPECS/wazuh-agent-aix.spec [hunk 2]
```diff
@@ -302,7 +314,7 @@ rm -fr %{buildroot}
 - More info: https://documentation.wazuh.com/current/release-notes/
 * Wed Oct 06 2021 support <info@wazuh.com> - 4.2.3
 - More info: https://documentation.wazuh.com/current/release-notes/
-* Wed Oct 06 2021 support <info@wazuh.com> - 4.2.2
+* Tue Sep 28 2021 support <info@wazuh.com> - 4.2.2
```
Comment (okynos) [inline aix/SPECS/wazuh-agent-aix.spec:317]: Fix 4.1.5, not Monday

**Review by okynos — CHANGES_REQUESTED** [review]
Minor changes

File: aix/SPECS/wazuh-agent-aix.spec (hunk 2 above)
Comment (rauldpm) [reply aix/SPECS/wazuh-agent-aix.spec:317]: Done in https://github.com/wazuh/wazuh-packages/pull/1677/commits/42feb16ca391c17786ee0ba99a222c2e06c4ec24

File: CHANGELOG.md (hunk 1 above)
Comment (rauldpm) [reply CHANGELOG.md:48]: Done in https://github.com/wazuh/wazuh-packages/pull/1677/commits/a50ea9847b32051d1fa02e721a6cfe2ffd557864

**Review by okynos — APPROVED** [review]

**Pull request merged**